### DISCUSSION TITLE
Call `layer_lock_set_user` from `layer_lock_all_off`

### DISCUSSION THREAD

**Discussion by doenietzomoeilijk** [getreuer/qmk-keymap#41] [Ideas]
Basically what it says on the tin. 

I was using the various `..._user` callbacks to drive content on my OLED displays, but this didn't work for layer locks when I had a timeout set on them. The lock would get dropped, but the OLED wouldn't reflect the new situation.

In my limited understanding of things, a simple `layer_lock_set_user(locked_layers);` after setting the locked_layers to 0 in `layer_lock_all_off` should also fire off that function when layers are dropped from the timer, is that correct?

**Comment by getreuer** [comment]
@doenietzomoeilijk thank you for catching this! You are exactly right, `layer_lock_all_off` should have but forgot to call the user callback, and this breaks updates to the OLED display.

I just pushed a fix in commit https://github.com/getreuer/qmk-keymap/commit/fb307bd456358955427c1874dfff9366eaee4eeb. Please pull or download again to get the latest.

**Comment by doenietzomoeilijk** [comment]
Excellent, thanks!
